CHANGELOG.md

```diff
@@ -5,6 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.70.0](https://github.com/buildkite/agent/tree/v3.70.0) (2024-04-18)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.69.0...v3.70.0)
+
+### Added
+- Add BUILDKITE_STEP_KEY as a json logger field [#2730](https://github.com/buildkite/agent/pull/2730) (@joeljeske)
+- New flag `--spawn-per-cpu` The number of agents to spawn per cpu in parallel (mutually exclusive with --spawn) [#2711](https://github.com/buildkite/agent/pull/2711) (@mmlb)
+- Upload agent images to GHCR [#2724](https://github.com/buildkite/agent/pull/2724) (@DrJosh9000)
+
```

Comment:
Recently we've been grouping all the dependabot churn under `### Internal`, as a single dot point (but I don't really mind if we have several dot points)